app/(auth)/change-password/page.tsx

```diff
@@ -63,27 +65,53 @@ export default function ChangePassword() {
   )
 
   return (
-    <>
-      <Heading>Change Password</Heading>
-      <Box as="form" onSubmit={changePasswordHandler}>
-        <Flex flexDirection={'column'}>
+    <Box
+      as="main"
+      w="100vw"
+      minH={{ base: 'calc(100vh - 145px)', md: 'calc(100vh - 210px)' }}
+      color={color}
+    >
+      <VStack
+        mt={{ base: '20px', md: '40px' }}
+        mx="auto"
+        w={{ base: '380px', md: '480px' }}
+        px={{ base: '16px', md: '80px' }}
+      >
```

Comment:
Padding should be outside of the form? 
Also you need space 70px below the button like the Figma design
<img width="1238" alt="Screenshot 2024-02-04 at 11 33 49 AM" src="https://github.com/tabi-memo/frontend/assets/66394413/5083ccda-6fd0-4f0c-a125-661e1bd7f2cd">

Reply:
Thank you for catching that! I added 70px paddings under the button and added left and right padding outside of the form!

Reply:
<img src="https://github.com/tabi-memo/frontend/assets/70562492/e1b552c6-4908-49b3-9758-f4e0fdfc4e9a" alt="Screenshot 1" width="300">
<img src="https://github.com/tabi-memo/frontend/assets/70562492/76c38aa8-52bf-4a24-8872-6abd856e259f" alt="Screenshot 2" width="300">